.github/workflows/rspec-package-specs.yml

```
@@ -47,6 +47,9 @@ jobs:
         git config user.email "you@example.com"
         git config user.name "Your Name"
         git commit -am "stop generators from complaining about uncommitted code"
+    - name: Set packer version environment variable
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:82: Double quote to prevent globbing and word splitting

(shellcheck)